components/automate-ui/src/app/page-components/profile/profile.component.scss

```diff
@@ -82,6 +82,11 @@ a, button:not(.dropdown-toggle) {
   &:first-child {
     border-bottom: 1px solid $chef-grey;
   }
+
+  .version,
+  .build {
+    line-height: 24px;
```

Comment:
The span's still required a line-height to apply the required spacing.  Line-height that is double the font-size seems consistent with other elements on the page